Commands/ascii.py

```
@@ -7,40 +7,54 @@
-    
+
     # Custom help argument to avoid conflict with -h for height
-    parser.add_argument('--help', action='help', help="Show this help message and exit.")
+    parser.add_argument('--help', action='help',
+                        help="Show this help message and exit.")
 
     try:
         parsed_args = parser.parse_args(args)
         return vars(parsed_args)
     except argparse.ArgumentError as e:
         raise argparse.ArgumentError(None, str(e))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)